test/gui/config.ini

```diff
@@ -1,5 +0,0 @@
-[DEFAULT]
-CLIENT_SYNC_PATH=
-BACKEND_HOST=http://localhost/owncloud-core/
-CLIENT_SYNC_TIMEOUT=
-MIDDLEWARE_URL=
```

Comment:
Why delete this?

Reply:
We already have config.sample.ini and this file is supposed to be different with respect to local setup.